File: fake/s390x.c
__asm__(".symver clock_getres,__kernel_clock_getres@@LINUX_2.6.29");
void clock_getres(void) {}
__asm__(".symver clock_gettime,__kernel_clock_gettime@@LINUX_2.6.29");
void clock_gettime(void) {}
__asm__(".symver gettimeofday,__kernel_gettimeofday@@LINUX_2.6.29");
void gettimeofday(void) {}

File: defs/s390x.rs
/// vDSO for `s390x`
#[derive(Debug, Copy, Clone)]
pub struct Vdso {
    pub clock_getres:  __kernel_clock_getres    @ "LINUX_2.6.29",
    pub clock_gettime: __kernel_clock_gettime   @ "LINUX_2.6.29",
    pub gettimeofday:  __kernel_gettimeofday    @ "LINUX_2.6.29",
}
